app.js

```diff
@@ -41,12 +44,26 @@ app.use(cors());
 app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
+app.use(cookieParser()); // DO NOT MODIFY THIS LINE, IT MAY SCREW UP EXPRESS-SESSION
+if (env == 'production' || env == 'test') {
+    var RedisStore = require('connect-redis')(session);
+    var redisClient = redis.createClient();
+}
+app.use(
+    session({
+        store:
+            env == 'production' || env == 'test'
+                ? new RedisStore({ client: redisClient })
+                : undefined,
+        secret: 'groupin-session-salt-very-very-secret',
```

Comment:
this shouldn't be exposed as publicly as well, right?

Reply:
it's not a third party API secret.
i'v changed it so in production we take the secret from the environment